app/notifications/process_client_response.py

```diff
@@ -83,5 +84,7 @@ def process_sms_client_response(status, reference, client_name):
 
     create_outcome_notification_statistic_tasks(notification)
 
```

Comment:
We could check for the existence of service_callback_api before creating the task.

Reply:
changed in 3594fc.